index.js

```diff
@@ -298,21 +281,21 @@ function buildOperation(oldOperation, produces, consumes, resourcePath) {
     });
   }
 
-  if (oldOperation.type && oldOperation.type !== 'void' &&
-      primitiveTypes.indexOf(oldOperation.type) === -1) {
-    operation.responses['default'] = {
-      'schema': {
-        '$ref': '#/definitions/' + oldOperation.type,
-      }
+  if (!Object.keys(operation.responses).length ||
```

Comment:
`Object.keys` throws if `operation.responses` is not an object. Please check for that before using `Object.keys`